### DISCUSSION TITLE
indonesian 是哪个字典？没有找到id 的

### DISCUSSION THREAD

**Discussion by cqray1990** [PaddlePaddle/PaddleOCR#13126] [Q&A]
#### 运行环境 / Runtime Environment
- OS:
- Paddle:
- PaddleOCR:

**Comment by UserWangZz** [comment]
可以具体描述一下你的问题吗？

**Comment by cqray1990** [comment]
> 可以具体描述一下你的问题吗？
@UserWangZz 
印尼语识别的字典的是哪个？，没有找到，看缩写是id_dict.txt,但是没找到

**Comment by UserWangZz** [comment]
印尼语的字典可能和其他语种共用了

**Reply by cqray1990** [reply]
> 印尼语的字典可能和其他语种共用了

印尼的对应哪个模型呢？
